db/schema.rb

```diff
@@ -53,17 +49,22 @@
     t.boolean "done"
```

Comment:
What's the purpose of this one? I did not find it on spreadsheet, maybe we can delete it.